job/models.py

```diff
@@ -5623,6 +5623,280 @@ class CoolingLoadOutputs(BaseModel, models.Model):
+    )    
+
+    installed_cost_ghx_per_ft = models.FloatField(
+        default=14.0,
+        validators=[
+            MinValueValidator(0.0),
+            MaxValueValidator(100.0)
+        ],
+        blank=True,
+        null=True,
+        help_text="Installed cost of the ground heat exchanger (GHX) in $/ft of vertical piping"
+    ) 
+
+    installed_cost_building_hydronic_loop_per_sqft = models.FloatField(
+        default=1.70,
+        validators=[
+            MinValueValidator(0.0),
+            MaxValueValidator(100.0)
+        ],
+        blank=True,
+        null=True,
+        help_text="Installed cost of the building hydronic loop per floor space of the site"
+    ) 
+
+    om_cost_per_sqft_year = models.FloatField(
+        default=-0.51,
+        validators=[
+            MinValueValidator(-100.0),
+            MaxValueValidator(100.0)
+        ],
+        blank=True,
+        null=True,
+        help_text="Annual GHP incremental operations and maintenance costs in $/ft^2-building/year"
+    ) 
+
+    # REQUIRED FOR GHP
+    building_sqft = models.FloatField(
+        validators=[
+            MinValueValidator(0.0),
+            MaxValueValidator(MAX_BIG_NUMBER)
+        ],
+        help_text="Building square footage for GHP/HVAC cost calculations"
+    ) 
+
+    space_heating_efficiency_thermal_factor = models.FloatField(
+        validators=[
+            MinValueValidator(0.001),
+            MaxValueValidator(1.0)
+        ],
+        blank=True,
+        null=True,
+        help_text="Heating efficiency factor (annual average) to account for reduced space heating thermal load from GHP retrofit (e.g. reduced reheat)"
+    )    
+
+    cooling_efficiency_thermal_factor = models.FloatField(
+        validators=[
+            MinValueValidator(0.001),
+            MaxValueValidator(1.0)
+        ],
+        blank=True,
+        null=True,
+        help_text="Cooling efficiency factor (annual average) to account for reduced cooling thermal load from GHP retrofit (e.g. reduced reheat)"
+    )
+
+    ghpghx_inputs = ArrayField(
+        models.JSONField(
+            null=True,
+            editable=True
+        ),
+        default=list, blank=True, null=True,
+        help_text=("GhpGhx.jl inputs/POST to ghpghx app")
+    )
+
+    # TODO do we need this? What would it do? Need to process this into ghpghx_responses...
+    ghpghx_response_uuids = ArrayField(
+        models.TextField(
+            blank=True
+        ),
+        default=list, blank=True, null=True,
+        help_text=("List of ghp_uuid's (like run_uuid for REopt) from ghpghx app, used to get GhpGhx.jl run data")    
+    )
+
+    ghpghx_responses = ArrayField(
+        models.JSONField(
+            null=True,
+            editable=True
+        ),
+        default=list, blank=True, null=True,
+        help_text=("ghpghx app response(s) to re-use a previous GhpGhx.jl run without relying on a database entry")
+    )
```

Comment:
Good call, I added handling of ghpghx_response_uuids in 0ea7fb4d8ba1c27d6af611c641fabae0c17bdc83. I tested this locally by creating a ghp_uuid entry in the /ghpghx database, and then I POSTed a run to /dev with GHP.ghpghx_response_uuids, and it worked. I had to delete the ghpghx_response_uuids from the POST because it was an unexpected argument for the GHP object in REopt.jl, and while ideally it would still be recorded in the inputs, I think this is fine for now. In theory I would add a unit test for this, but that would get a little tricky because the test suite uses a temporary database so the /ghpghx uuid would have to be created first and somehow stored before the test could be run. This is doable, but since this is a pretty rare use case, it's not worth the effort right now.

Reply:
This is exactly the test case and addition to `clean()` I was thinking of - thank you!  I think that given the specific use cases intended for GHP I agree this is sufficient, and we can revisit if an issue comes up from users trying to do this for some reason.